README.md

```diff
@@ -1,48 +1,55 @@
-# openEO Project Steering Committee Guidelines
+# openEO Project Steering Committee
 
-Author: Edzer Pebesma, mostly copied from https://gdal.org/development/rfc/rfc1_pmc.html#rfc-1
+This document describes how the openEO Project Steering Committee determines membership, and makes decisions on openEO project issues. The openEO project is concerned with all repositories (software and specifications etc.) found under https://github.com/Open-EO.
+
+In brief the committee votes on proposals on https://github.com/Open-EO/PSC/issues. Proposals are available for review for at least two business days, and a single veto is sufficient to delay progress though ultimately a majority of members can pass a proposal.
 
-Contact: edzer.pebesma@uni-muenster.de
+## Members
 
-Status: Proposed
+Matthias Mohr is declared initial Chair of the openEO Project Steering Committee. The initial list of members of the Project Steering Committee is:
 
-## Summary
+* Jeroen Dries (@jdries)
+* Luca Foresta (@lforesta)
+* Alexander Jacob (@aljacob)
+* Miha Kadunc (@mkadunc)
+* Matthias Mohr (**chair**, @m-mohr)
+* Markus Neteler (@neteler)
+* Edzer Pebesma (@edzer)
 
-This document describes how the openEO Project Steering Committee determines membership, and makes decisions on openEO project issues. The openEO project is concerned with all software found under https://github.com/Open-EO.  
-In brief the committee votes on proposals on https://github.com/Open-EO/PSC. Proposals are available for review for at least two (German) business days, and a single veto is sufficient to delay progress though ultimately a majority of members can pass a proposal.
+The current list will be maintained by the openEO chair on this page and will be published to the openEO web pages, too.
```

Comment:
Changed this. Should be maintained here, I'll implement that this will be automatically included in the homepage.